css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,319 @@
+	with a true condition.
+
+	<div class=example>
+		For example, ''@sheet'' and ''@media'' can be mixed:
+
+		<pre class=lang-css>
+		@sheet framework {
+			h1, h2 { color: maroon; background: white;}
+
+			@media (prefers-color-scheme: dark) {
+				h1, h2 { color: red; background: black; }
+			}
+		}
+		</pre>
+	</div>
+
+	Note: ''@sheet'' [=block at-rules=] cannot be interleaved with ''@import'' rules.
+
+<h3 id="sheet-declaration">
+Declaring Named Stylesheets</h3>
+
+	Named stylesheets can be declared:
+
+	* using an ''@import'' rule with the ''from'' keyword
+		assigning the contents of the imported file into that sheet.
+	* using an [[#at-ruledef-sheet|@sheet block at-rule]],
+		assigning its child style rules into that sheet.
+
+<h3 id="at-ruledef-sheet-block">
+Subdividing Stylesheets: the `@sheet` block at-rule</h3>
+
+<pre class='prod'>
+		@import [ <<string>> from ]?
+				[ <<url>> | <<string>> ]				
+		        [ layer | layer(<<layer-name>>) ]?
+		        <<import-conditions>> ;
+
+		<dfn export>&lt;import-conditions></dfn> = [ supports( [ <<supports-condition>> | <<declaration>> ] ) ]?
+		                     <<media-query-list>>?</pre>
+
+	where:
+	* <<string>> ''from'' imports only the named sheet
+		with name <<string>> from the provided URL.
+
+	* the <<url>> or <<string>>
+		gives the URL of the style sheet to be imported.
+
+	* the optional ''layer'' keyword or ''layer()'' function
+		assigns the contents of the style sheet
+		into its own anonymous [=cascade layer=]
+		or into the named [=cascade layer=].
+
+		The layer is added to the layer order
+		even if the import fails to load the stylesheet,
+		but is subject to any [=import conditions=]
+		(just as if declared by an ''@layer'' rule wrapped
+		in the appropriate [=conditional group rules=]).
+
+	* the optional <<import-conditions>>
+		states the [=import conditions=] under which it applies.
+
+	<div class="example">
+		The following <a href="#conditional-import">conditional <css>@import</css> rule</a>
+		only loads the style sheet with a name of ``foo`` from ``narrow.css`` when the UA
+		<a href="https://www.w3.org/TR/css-conditional-3/#support-definition">supports</a> ''display: flex'',
+		and only applies the style sheet on a <a href="https://www.w3.org/TR/CSS2/media.html#media-types">handheld</a> device
+		with a <a href="https://www.w3.org/TR/mediaqueries-4/#width">maximum viewport width</a> of 400px.
+
+		<pre>@import foo from url("narrow.css") supports(display: flex) handheld and (max-width: 400px);</pre>
+	</div>
+
+	<div class="example">
+		The following layer imports load the style sheets named ``foo`` into
+		the ''framework.component'' layer, and an un-named layer, respectively:
+
+		<pre>
+		@import foo from url("tabs.css") layer(framework.component);
+		@import foo from url("override.css") layer;
+		</pre>
+	</div>
+
+	If a <<string>> is provided,
+	it must be interpreted as a <<url>> with the same value.
```

Comment:
Related to my comment above about `<url-string>` - this text should also be associated with that definition, since it only applies there and not to the `<string>` used with "from".

Reply:
I might also bring the example below into the same context for `<url-string>`.